custom_components/tuya_local/devices/jiahong_et72w_thermostat.yaml

```diff
@@ -113,6 +105,9 @@ entities:
             value: C
           - dps_val: true
             value: F
+      - id: 113
+        type: integer
+        name: room_temp_calibration
```

Comment:
Since this is attached now to the Room temperature sensor, the attribute can just be "calibration". I am not sure if it might be worth splitting this out to a number entity though, as this is usually for the user to adjust based on compensation for the installation location, unlike the calibration options for smartswitches which are more for factory calibration and take specialist equipment to tune.

Reply:
Yes, having calibration dps as config sensors makes sense. I was planning to do that in a seperate PR.